kaldb/src/main/java/com/slack/kaldb/elasticsearchApi/ElasticsearchApiService.java

```diff
@@ -47,6 +53,9 @@
 public class ElasticsearchApiService {
   private static final Logger LOG = LoggerFactory.getLogger(ElasticsearchApiService.class);
   private final KaldbQueryServiceBase searcher;
+  private final ExecutorService executorService =
```

Comment:
Please add a comment on why we create our own pool for better tracing so it lives with the code.

Reply:
Done!